installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc

```diff
@@ -0,0 +1,50 @@
+[id="installing-vsphere-installer-provisioned"]
+= Installing a cluster on vSphere
+include::modules/common-attributes.adoc[]
+:context: installing-vsphere-installer-provisioned
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on your
+VMware vSphere instance using installer-provisioned infrastructure.
+
+.Prerequisites
+
+* Provision
+xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage]
+for your cluster. To deploy a private image registry, your storage must provide
+ReadWriteMany access modes.
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* If you use a firewall, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/installation-vsphere-infrastructure.adoc[leveloffset=+1]
+
+include::modules/installation-vsphere-installer-infra-requirements.adoc[leveloffset=+1]
+
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/installation-launching-installer.adoc[leveloffset=+1]
```

Comment:
before launching installer we should tell users they need to trust the vcenter CA like this: https://github.com/openshift/installer/blob/master/docs/user/vsphere/install.md#trust-vcenter-api